R/recipe_run.R

```diff
@@ -381,8 +381,7 @@ recipe_check_artefacts <- function(info) {
   }
   unexpected <- recipe_unexpected_artefacts(info, NULL)
   if (length(unexpected) != 0) {
-    orderly_log("unex_art", paste("Unexpected artefacts created:",
-                                  unexpected, collapse = ", "))
+    orderly_log("unexpect", sprintf("%s", unexpected))
```

Comment:
It's really disappointing that `unexpected` does not fit here.  Can you tweak `orderly_log` to make it fit (this will likely break a few tests).  Otherwise can we use `extra` instead?